src/components/PetForm/PetForm.js

```diff
@@ -269,6 +289,7 @@ export default class PetForm extends Component {
           />
           <label className='form-check-label' htmlFor='houseTrained'>House Trained</label>
         </FormGroup>
+        <FilesUploadComponent id='picture' onChange={this.onFileChange} required />
```

Comment:
I feel like the image upload gets lost under the checkboxes. What do you think about moving it up to somewhere around the description? There should also be a `<FormGroup>` around this element for consistency, and it should be removed from the `FilesUploadComponent` class

Reply:
That's a good idea. I moved it under description.
![image](https://user-images.githubusercontent.com/71732016/153918573-d4833a81-9439-4382-b97e-073eb3953a05.png)